### PR TITLE
Added dependency security checker into composer

### PR DESCRIPTION
**Pull request by Skyld**
- Added dependency security checker into composer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,5 +76,5 @@
 ## 1.1.0 - 2019-03-19
 
 ### Added
-- Added possibility to specify charset for regular and native ingesters.
+- Added possibility to specify charset for regular and native ingester.
```
Comment (luisfn) [inline CHANGELOG.md]: Not sure why not use the previous plural, since it's related to the 2 types (regular/native).

File: config/bundles.php [hunk 2]
```diff
@@ -28,12 +10,12 @@
...
     Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
-    Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle::class => ['all' => true],
     Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
     Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'test' => true],
     Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle::class => ['dev' => true, 'test' => true],
     Fidry\AliceDataFixtures\Bridge\Symfony\FidryAliceDataFixturesBundle::class => ['dev' => true, 'test' => true],
     Hautelook\AliceBundle\HautelookAliceBundle::class => ['dev' => true, 'test' => true],
     Nelmio\CorsBundle\NelmioCorsBundle::class => ['all' => true],
+    Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle::class => ['all' => true],
```
Comment (luisfn) [inline config/bundles.php]: As curiosity, why this order change?

File: CHANGELOG.md (hunk 1 above)
Comment (Skyld) [reply CHANGELOG.md]: Mmmmm, maybe you are right, reverted.

File: config/bundles.php (hunk 2 above)
Comment (Skyld) [reply config/bundles.php]: This file automatically updated by Symfony. I removed the doctrine cache bundle from terminal then I re-added it (I was testing something) and it appended it at the end of the list. The order of the bundles here is not relevant.

**Pull request merged**